### PR TITLE
Added OptionsSelect component

### PR DESCRIPTION
**Pull request by VD39**

### CODE REVIEW

**Comment by VD39** [comment]
Reference: #154

File: src/components/05_pages/NodeForm/RecipeSchema.js [hunk 1]
```diff
@@ -133,8 +133,8 @@ export default {
         field_difficulty: {
           type: 'string',
           title: 'Difficulty',
-          enum: ['easy', 'medium', 'hard'],
-          default: 'medium',
+          options: ['easy', 'medium', 'hard'],
+          defaultValue: 'medium',
```
Comment (VD39) [inline src/components/05_pages/NodeForm/RecipeSchema.js]: Change values, as ones used are reserved, so using them is not possible when deconstructing objects.

Comment (dawehner) [reply src/components/05_pages/NodeForm/RecipeSchema.js]: Mh, I fear we will not be able to change this output. This is a copy of what will come back from openapi.

Comment (VD39) [reply src/components/05_pages/NodeForm/RecipeSchema.js]: Change made and I also updated the component.

File: src/components/02_atoms/Widgets/OptionsSelect.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  `,
+};
+
+const OptionsSelect = ({
+  label,
+  value,
+  onChange,
+  helpText,
+  fieldName,
+  enum: options,
```
Comment (dawehner) [inline src/components/02_atoms/Widgets/OptionsSelect.js]: Mh, this doesn't work yet because I guess we don't pass the schema along, yet

Comment (dawehner) [reply src/components/02_atoms/Widgets/OptionsSelect.js]: Gotcha. I think we should pass along the schema of the field to each widget. This way you could extract the information you need. Does that make sense?

Comment (dawehner) [reply src/components/02_atoms/Widgets/OptionsSelect.js]: Do you want to do this as part of this PR?

Comment (VD39) [reply src/components/02_atoms/Widgets/OptionsSelect.js]: Sure, I can do that.

File: src/components/02_atoms/Widgets/OptionsSelect.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  schema: PropTypes.shape({
+    default: PropTypes.string,
+    enum: PropTypes.arrayOf(PropTypes.string),
+  }),
+};
+
+OptionsSelect.defaultProps = {
+  value: '',
+  helpText: '',
+  schema: {
```
Comment (dawehner) [inline src/components/02_atoms/Widgets/OptionsSelect.js]: I do have one question: Why would you provide a default schema, which isn't really helpful. I'd rather suggest to make it required, so you cannot use it wrong.

Comment (VD39) [reply src/components/02_atoms/Widgets/OptionsSelect.js]: Ah, fair point. Change has been made.

**Pull request merged**